README.md

```diff
@@ -21,9 +21,36 @@ A microservice for storing associations between user IDs and DIDs for applicatio
       ````
 * Run tests, build jar, and run DID Mapping using:
     ````bash
-  mvn package
+  mvn
   java -jar target/did-mapping-1.0-SNAPSHOT.jar 
   ````
+* Regenerate swagger json after modifications to the REST API or models
+    ````bash
+  mvn -Pwrite-swagger-json
```

Comment:
Hmzz, not liking this being a separate command now. Rather would see that as part of the build process in the generate sources stage. This will allow for stale end-results because somebody forgot to run the command

Reply:
I was / am actually struggling in maven to execute the specific order of plugins
1. Run the surefire plugin on WriteSwaggerJson unit test to generate the swagger.json
2. Run the swagger2markup-maven-plugin which consumes the generated swagger.sdk.json
3. Run the surefire plugin again to do the unit tests

The simplest way to achief this is to run two profiles in sequence. 
There is also a kongchen-swagger-maven-plugin which claims it can generate the swagger.json, but I don't think it's a good solution because it doesn't spin up an instance using our RestTemplates and apiInfo and doesn't support the store vs sdk modes that we use.
The cleanest way would be to write a new maven plugin wich can replace WriteSwaggerJson for all our projects and can handle the exceptions we do based on "Mode". But that's a whole other job.

Reply:
It may also be possible with multipule executions on the surefire-plugin and let WriteSwaggerJson run in the process-test-resourcesor generate-test-resources phase.
But I would need to expirment with that, executions don't always work.

Reply:
The latter is the better solution.

Reply:
I was able to fix this. Not with multiple executions because the surefire plugin only binds to the test phase which is too late.
However exec-maven-plugin can bind to the generate-test-resources phase and insert a "mvn -Pwrite-swagger-json" right before the test phase starts.